ubuntu/python/Dockerfile

```diff
@@ -1,25 +1,36 @@
 FROM databricksruntime/minimal:10.4-LTS
 
+# These are the versions compatible for DBR 9.x and 10.x
```

Comment:
Is this comment relevant for both PRs? don't we just need 9.x for the 9 LTS, and 10.x for the 10 LTS?

Reply:
ah yes. internally the python and library versions related to python env are the same between 9.x and 10.x LTS. we can made this comment only relevant to just one version if needed

Reply:
If it's easy to, let's make them consistent with the intended LTS to avoid confusion amongst users.